metrics-server.yaml

```
@@ -36,10 +36,26 @@ pipeline:
 
   - uses: strip
 
-# Note, we don't need to fetch any upstream files, as there aren't any custom
-# entrypoints or scripts required for compatibility with the metrics-server
-# Bitnami helm chart.
 subpackages:
+  - name: ${{package.name}}-compat
+    pipeline:
+      - working-directory: ${{targets.subpkgdir}}
+        runs: |
```

```suggestion
      - runs: |
```
Nit: working-directory can be omitted here.